### ISSUE TITLE
Keyring does not support prompts

### ISSUE THREAD

**Issue by jaraco** [jaraco/keyring#98]
I'm getting this when using mercural keyring on fedora 18 i686

```
 File "/usr/lib/python2.7/site-packages/keyring/backends/SecretService.py", line 76, in _check_prompt
    raise ValueError("Keyring does not support prompts")
ValueError: Keyring does not support prompts
```

---
- Bitbucket: https://bitbucket.org/kang/python-keyring-lib/issue/98
- Originally reported by: Mitchel  Kelonye
- Originally created at: 2013-03-23T16:17:27.825

**Comment by jaraco** [comment]
Please close this issue, it is fixed since 1.3.

---

Original comment by: Dmitry Shachnev

**Comment by jaraco** [comment]
Ok, well decided to use ssh.

---

Original comment by: Mitchel  Kelonye
